3723/3/app/controllers/place_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PlaceController < ApplicationController
+  post '/places/:id' do
+    @comment = Comment.new(
+      title: params[:title],
+      rating: params[:rating],
+      place_id: params[:id],
+      user_id: current_user.id
+    )
+    @place = Place.find_by_id(params[:id])
+    if @comment.valid?
+      @comment.save
+      @place.update(rating: @place.comments.average(:rating))
```

Comment:
The code `@place.comments.average(:rating)` will looks better as method of model `Place`